nvi-commons/src/test/java/no/sikt/nva/nvi/common/service/CandidateTest.java

```diff
@@ -285,96 +247,15 @@ void shouldReturnGlobalApprovalStatusDisputeWhenConflictsExist() {
-        var createRequest = createUpsertCandidateRequest(randomUri(), randomUri(), true,
-                                                         InstanceType.ACADEMIC_MONOGRAPH, 4, totalPoints,
-                                                         randomLevelExcluding(DbLevel.NON_CANDIDATE)
-                                                             .getValue(), CURRENT_YEAR,
-                                                         institutionToApprove, randomUri(), institutionToReject);
-        var candidateBO = upsert(createRequest);
-        candidateBO.createNote(createNoteRequest(randomString(), randomString()), candidateRepository)
-            .createNote(createNoteRequest(randomString(), randomString()), candidateRepository)
-            .updateApproval(createUpdateStatusRequest(ApprovalStatus.APPROVED, institutionToApprove, randomString()))
-            .updateApproval(createUpdateStatusRequest(ApprovalStatus.REJECTED, institutionToReject, randomString()));
-        var dto = candidateBO.toDto();
-        var approvalMap = dto.approvals()
-                              .stream()
-                              .collect(Collectors.toMap(ApprovalDto::institutionId, Function.identity()));
-
-        assertAll(() -> {
-            assertThat(dto.context(), is(equalTo(CONTEXT_URI)));
-            assertThat(dto.publicationId(), is(equalTo(createRequest.publicationId())));
-            assertThat(dto.approvals().size(), is(equalTo(createRequest.institutionPoints().size())));
-            assertThat(dto.notes().size(), is(2));
-            assertThat(dto.totalPoints(), is(equalTo(adjustScaleAndRoundingMode(totalPoints))));
-            var note = dto.notes().getFirst();
-            assertThat(note.text(), is(notNullValue()));
-            assertThat(note.user(), is(notNullValue()));
-            assertThat(note.createdDate(), is(notNullValue()));
-            assertThat(dto.id(), is(equalTo(constructId(candidateBO.getIdentifier()))));
-            assertThat(dto.identifier(), is(equalTo(candidateBO.getIdentifier())));
-            var periodStatus = getDefaultPeriodStatus();
-            assertThat(dto.period().status(), is(equalTo(periodStatus.status())));
-            var approvalDto = approvalMap.get(institutionToApprove);
-            assertThat(approvalDto.status().getValue(), is(equalTo(ApprovalStatusDto.APPROVED.getValue())));
-            assertNotNull(approvalDto.finalizedDate());
-            var rejectedAP = approvalMap.get(institutionToReject);
-            assertThat(rejectedAP.status(), is(equalTo(ApprovalStatusDto.REJECTED)));
-            assertThat(rejectedAP.reason(), is(notNullValue()));
-            assertThat(rejectedAP.points(),
-                       is(adjustScaleAndRoundingMode(
-                           createRequest.getPointsForInstitution(rejectedAP.institutionId()))));
-        });
-    }
-
-    @Test
+        //These getters are tested in different modules, so this is just for jacoco coverage
```

Comment:
This comment looks a bit out of place now. It makes sense in the context of the PR, but does it belong here otherwise?